devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -1,12 +1,9 @@
-/**
- *  Keen Home Smart Vent
- *
- *  Author: Keen Home
- *  Date: 2015-06-23
- */
+// keen home smart vent
+// http://www.keenhome.io
+// SmartThings Device Handler v1.0.0
 
 metadata {
-    definition (name: "Keen Home Smart Vent", namespace: "Keen Home", author: "Gregg Altschul") {
+    definition (name: "Keen Home Smart Vent", namespace: "keenhome", author: "Keen Home") {
```

Comment:
@workingmonk  Do you know how this will affect everything? It looks like the path and namespace won't match. I suspect regardless only new installs will get the new code, and existing devices will not.